webui/react/src/components/ExperimentActionDropdown.tsx

```diff
@@ -85,12 +96,57 @@ const ExperimentActionDropdown: React.FC<Props> = ({
   const ExperimentEditModal = useModal(ExperimentEditModalComponent);
   const ExperimentMoveModal = useModal(ExperimentMoveModalComponent);
   const ExperimentRetainLogsModal = useModal(ExperimentRetainLogsModalComponent);
-  const HyperparameterSearchModal = useModal(HyperparameterSearchModalComponent);
+  const {
+    Component: HyperparameterSearchModal,
+    open: hyperparameterSearchModalOpen,
+    close: hyperparameterSearchModalClose,
+  } = useModal(HyperparameterSearchModalComponent);
+  const {
+    Component: InterstitialModal,
+    open: interstitialModalOpen,
+    close: interstitialModalClose,
+  } = useModal(InterstitialModalComponent);
+  const [experimentItem, setExperimentItem] = useState<Loadable<FullExperimentItem>>(NotLoaded);
+  const canceler = useRef<AbortController>(new AbortController());
```

Comment:
where does this call abort?

Reply:
in `onInterstitalClose` func. When user click close button in loading modal, cancel API

Reply:
got it!